go/simple/v1/ampli/ampli.go

```diff
@@ -0,0 +1,385 @@
+// ampli.py
+//
+// Ampli - A strong typed wrapper for your Analytics
+//
+// This file is generated by Amplitude.
+// To update run 'ampli pull go-ampli'
+//
+// Required dependencies: analytics-go
+// Tracking Plan Version: 0
+// Build: 1.0.0
+// Runtime: go-ampli
+//
+// [View Tracking Plan]: https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest
+//
+// [Full Setup Instructions]: https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/main/latest/getting-started/go-ampli
+//
+
+package ampli
+
+import (
+	"log"
+
+	"github.com/amplitude/analytics-go/amplitude"
+)
+
+var Client = Ampli{}
+
+type Environment string
+
+const (
+	DEVELOPMENT Environment = "development"
```

Comment:
Go style is `EnvironmentDevelopment` and `EnvironmentProduction`

Reply:
renamed